common/source/docs/common-matek-m8q.rst

```diff
@@ -60,10 +60,10 @@ The default airspeed sensor is expected to be a "DLVR-L10D" type , if not then y
 
 if using an:
 
-- MS4525    set ARSP_TYPE -> 1,
-- MS5525    set ARSP_TYPE -> 5 (note: only an MS5525 with address 0x77 can be used to avoid conflict with DPS310 barometer.)
-- SDP3X     set ARSP_TYPE -> 6
-- DLVR-L10D set ARSP_TYPE -> 9
+- MS4525    set ARSPD_TYPE -> 1,
```

Comment:
we should make these param links...ie :ref:`ARSPD_TYPE<ARSPD_TYPE>`

Reply:
Done